config/initializers/health_check.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+HealthCheck.setup do |config|
+  # uri prefix (no leading slash)
+  config.uri = "health_check"
```

Comment:
We'd usually want this dasherised rather than snake cased. But this is ok as this isn't quite user facing.

Reply:
I think I will change it to one word - Matt gave me an example of other services which don't use hyphens and use the word healthcheck https://pp-services.signin.education.gov.uk/healthcheck

Reply:
changed to healthcheck